### PR TITLE
Add Table of Contents 

### PR DESCRIPTION
**Pull request by nuyer**
Continue from Jieling's User Guide PR
- add Table of content hyperlink to each section
- change manager -> HR and relations Manager

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -111,7 +142,8 @@ Examples:
 
 **Warnings:** Due to the destructive nature of this action, NetConnect will require a confirmation from the user before it is executed.
 
-### Adding a Remark to a Person : `remark`
+<section id="remark"/>
+## Adding a Remark to a Person : `remark`
```
Comment (shaunlxw) [inline docs/UserGuide.md]: Need a line between these.

**Review by shaunlxw — CHANGES_REQUESTED** [review]
LGTM! The table of contents looks sick 🔥 Not sure if it my platform issue, but using the link jumps to slightly below the section 😢

**Review by starsia — APPROVED** [review]
Looks good!

**Review by starsia — APPROVED** [review]

**Review by shaunlxw — APPROVED** [review]
LGTM!

**Pull request merged**